src/modules/order.ts

```diff
@@ -181,6 +212,11 @@ export namespace OrderModule {
     /** commission percents, input 10 for 10% */
     referralCommission?: number;
     referralKickback?: number;
+    /** leverage for new trades is defaulted to 3x.
```

Comment:
this context is unnecessary for the SDK imo, not relevant in a generic case as there is no 3x anywhere in the code. possibly lead to more confusion.